### DISCUSSION TITLE
API Docs should allow changing the base url for self-hosted Outline

### DISCUSSION THREAD

**Discussion by dserodio** [outline/outline#7466] [Ideas]
The [API docs](https://www.getoutline.com/developers) does not allow changing the "base URL," so when you generate an example command, it tries to access the Outline SaaS at getoutline.com. It should be possible to change the base URL to generate commands for self-hosted Outline.

**Comment by tommoor** [comment]
@hanspagel is this possible? 🤔

**Reply by hanspagel** [reply]
Oh, I check my GitHub notifications only twice a year and today is the day. Totally valid use case, let me create an issue in our repository: https://github.com/scalar/scalar/issues/3459

**Reply by hanspagel** [reply]
Submitted a PR: https://github.com/outline/openapi/pull/18
